documentation/README.md

```diff
@@ -38,7 +38,7 @@ If you'd like to have a local copy of PMM documentation, or are thinking about c
 
 2. Clone this repository.
 
-3. Change directory to `pmm-doc`.
+3. Change directory to `pmm/documentation`.
```

Comment:
Given that `documentation` is a root directory in the repo, how can this be true?

Reply:
good point, I will close this PR to cherry pick changes for PMM 3